composer.json

```diff
@@ -16,11 +16,11 @@
     }
   ],
   "require": {
-    "php": "~7.1.3||~7.2.0||~7.3.0",
+    "php": "^7.1",
```

Comment:
Is this right?

Reply:
That line change is from an older commit, "php": "^7.1", is in master branch

Reply:
@roopesalminen is "~7.1.3||~7.2.0||~7.3.0" correct one? That is in QA when in master it's "php": "^7.1". Is this PR now gonna change it back to "php": "^7.1" in QA what it might not be

@richard-madarasz